e2e/test/FaultInjection.cs

```diff
@@ -84,6 +84,11 @@ public static async Task ActivateFaultInjection(Client.TransportType transport,
 
             uint oldTimeout = deviceClient.OperationTimeoutInMilliseconds;
 
+            if (faultType.Equals(FaultInjection.FaultType_Throttle))
+            {
+                deviceClient.SetRetryPolicy(new NoRetry());
+            }
```

Comment:
Can you create a ICM and track this as workaround around that until service resolves problem around them ?